lib/husky_musher/tests/kiosk_flow.py

```diff
@@ -0,0 +1,195 @@
+import sys
+import unittest
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).parent.parent))
+print(sys.path)
```

Comment:
Looks like this print may be for debugging

Reply:
Thanks! Fixed.